.github/workflows/add-artifacts-to-release.yml

```diff
@@ -0,0 +1,88 @@
+name: Release artifacts
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish_libcorerust:
+    name: Build and upload libcorerust
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b
+        with:
+          ref: ${{ github.event.release.tag_name }}
+          fetch-depth: 0
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - name: Cache Gradle packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Run local core-rust docker build
+        run: ./gradlew :core-rust:buildRustForDocker -P rustBinaryBuildType="release"
+      - name: Upload libcorerust
+        run:
+          gh release upload ${{ github.event.release.tag_name }} --repo radixdlt/babylon-node ${{ github.workspace }}/core-rust/target/x86_64-unknown-linux-gnu/release/libcorerust.so --clobber
```

Comment:
Any reason why you have to resort to gh cli .  We could do this using the action `actions/upload-release-asset`

Reply:
Sadly I deleted the tries with this action. They ended up canceling due to file type and size when uploading the debug build. The release is much smaller and if we end up zipping it either way like in the discussion, I can use the github action again.

I mentioned this in the daily, but I guess it was not clear.

Reply:
Does not apply anymore since the library gets zipped now